### ISSUE TITLE
Inconsistent naming

### ISSUE THREAD

**Issue by honfika** [justcoding121/Titanium-Web-Proxy#207]
The method argument names in your code ususally starts with lowercase letter, but sometimes it starts with uppercased, for example:
        public ExplicitProxyEndPoint(IPAddress IpAddress, int Port, bool EnableSsl)

Would you mind if I create a pull request which changes the casing everywhere in method arguments to lowercase?

Similar issue with the property names, they should be uppercased, but for example:
        private CertificateManager certificateCacheManager { get; set; }

Would you mind if I create a pull request which changes the casing everywhere in property names to uppercase?

**Comment by justcoding121** [comment]
The code style I tried to follow is as follows.

Property names should be lowercase for private 
Property names should be Uppercase for public, internal or any other.

Argument names should be always lower.

You can create PR if you find inconsistency.

**Comment by honfika** [comment]
Ok, pull request cretead.
I changed all of the argument name first characters to lower case, and also changed some internal properties to uppper case.

**Comment by honfika** [comment]
I saw that you added this information to the wiki:
"Method names should be camel case for private Method names should be pascal case for public, internal or any other."

Are you sure about it?
I checked your existing code, and did not found any private method which starts with lowercase character, so all of the method names are in PascalCase:
just some existing example:
        private void OnAcceptConnection(IAsyncResult asyn)
        private void PrepareRequestHeaders(Dictionary<string, HttpHeader> requestHeaders, HttpWebClient webRequest)
        private void FixProxyHeaders(Dictionary<string, HttpHeader> headers)
        private void Dispose(Stream clientStream, CustomBinaryReader clientStreamReader,
        private void SetProxy(string hostname, int port, ProxyProtocolType protocolType)
        private void RemoveProxy(ProxyProtocolType protocolType)
        private void Refresh()

And I think this is normal, all the methods (including private) should have PascalCase names. So I think the wiki should be fixed.

Btw: Usually all the property names has PascalCase, too (Just check an open sourse MS project), but in this case your source code contains camalCase names, so I did not want to force you to change them (and also did not changed them in my pull request as you requested).

**Comment by justcoding121** [comment]
In any case I personally don't like anything that starts with underscore. I was thinking of removing those added recently :-)

Your observations are correct. I will update wiki. Not sure if all private methods in project are PascalCase, will have to fix otherwise. 

I would like to keep private properties and fields as lower case. When coding this will help to identify easily if we are working on a public property which user can change.

**Comment by honfika** [comment]
I'm sorry, thats my fault, I'll remove the undersores with my next pull request.
It was generated  with Reshaprer automatically. Now I configured the naming rules.

There is only 1 more field which start with underscore:
WinCertificateMaker._SharedPrivateKey, I'd like to rename it to sharedPrivateKey

I checked, and there is no private method which has camelCase name.

What about the local variables? They should be camelCase, and they are usually in camelCase.
May I create a pull request which changes all of them to camelCase?
For example from:
                var SignerCertificate = Activator.CreateInstance(typeSignerCertificate);
to
                var signerCertificate = Activator.CreateInstance(typeSignerCertificate);
